src/Microsoft.DotNet.Build.VSTSBuildsApi/project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "Newtonsoft.Json": "9.0.1",
+    "NETStandard.Library": "1.6.0"
+  },
+  "frameworks": {
+    "netstandard1.5": {
+      "imports": "dnxcore50"
```

Comment:
We want to only include it when needed, so try removing it--if it fails, it's a restore-time failure so it's hard to miss.